jenkins/install_and_run.sh

```diff
@@ -14,6 +14,21 @@ cd "`dirname $0`/.."
 
 ./install-deps.pl
 
+: ${PORT_BASE=20000}
+: ${PORT_COUNT=100}
+: ${BIND_HOST=localhost}
+
+export PORT_BASE
+export PORT_COUNT
+export BIND_HOST
+
 ./jenkins/kill_old_listeners.sh
 
-./run-tests.pl --bind-host ${BIND_HOST:-localhost} -O tap --all "$@" > results.tap
+export TOX_BIN=$WORKSPACE/.tox/py27/bin
```

Comment:
Yes, using WORKSPACE means that this script will also work when sytest is checked out as a subfolder in the synapse and dendron jenkins tests.